code/index.html

```diff
@@ -3,28 +3,162 @@
+              Vi kommer ta upp olika viktiga ämnen inom bland annat jobb, ekonomi, sälj, företagande, förhandling, livet som kvinna, jämställdhet etc.
+              Så om du letar efter din flock, dina bossbabes, ditt bollplank och dina systrar som också vill framåt och samtidigt vill lyfta dig på samma sätt som du lyfter dem - då har du kommit helt rätt!
+            </p>
+          </div>
+      </div>
 
-      <div class="question" id="section1">Section 1</div>
-      <div class="answer">
-        <p>Lorem ipsum...</p>
+      <div class="accordion-item">
+        <div class="accordion-item-header">
+          <h4>Vem kan vara med i EDGE?</h4>
+        </div>
+          <div class="accordion-item-body">
+            <p class="accordion-item-body-content">
+              EDGE här är i grunden ett affärsnätverk där vi ska utvecklas som både som affärskvinnor som privatpersoner. 
+              Det spelar ingen roll om du är anställd, egenföretagare, eller något annat. För var du är nu behöver ju inte definiera var du villl vara, eller var du kommer att vara i framtiden. 
+              Det viktiga är din inställning och öppenhet eftersom det här är ett nätverk för kvinnor som vet att 1 + 1 = 3; att vi tillsammans blir starkare! 
+            </p>
+          </div>
       </div>
 
-      <div class="question" id="section2">Section 2</div>
-      <div class="answer">
-        <p>Lorem ipsum...</p>
+      <div class="accordion-item">
+        <div class="accordion-item-header">
+          <h4> Hur ofta ses vi? </h4>
+        </div>
+          <div class="accordion-item-body">
+            <p class="accordion-item-body-content">
+              Vi kommer att träffas minst en gång i månaden. Höstens planering kommer dyka upp här på webbplatsen inom kort. 
+              Stay tuned för mer info! 
+            </p>
+          </div>
       </div>
-    </section>
-    <!-- dont forget to link the javascript file here! -->
+
+      <div class="accordion-item">
+        <div class="accordion-item-header">
+          <h4>Vad kostar ett medlemsskap?</h4>
+        </div>
+          <div class="accordion-item-body">
+            <p class="accordion-item-body-content">
+              Ett årsmedlemsskap i EDGE kostar 500 kr, prova på i en månad för 150 kr.
+            </p>
+          </div>      
+        </div>
+    </div>
+    <script src="./script.js"></script> <!-- Link for JavaScript file -->
+    <br>
+    <br>
+
+    <!-- Signup form -->
+    <h2>KONTAKT</h2>
+    
+    <div class="container">
+      <form action="http://httpbin.org/anything" method="POST">
+        <div class="row">
+          <div class="col-25">
+            <label for="fname">Förnamn</label>
+          </div>
+          <div class="col-75">
+            <input type="text" id="fname" name="fname" placeholder="Förnamn">
+          </div>
+        </div>
```

Comment:
There's no need for using a class wrapped around the label or input since you can put classes in each of those, this will save you a lot fo code.